xml/ha_troubleshooting.xml

```diff
@@ -129,8 +129,14 @@
     <term>Where to find the log files? </term>
     <listitem>
      <para>
+      <remark>taroth 2018-03-05: ygao, should I replace the following with
+     "Pacemaker writes its logs to '/var/log/pacemaker/pacemaker.log' and
+     '/var/log/pacemaker/bundles/*/pacemaker.log, respectively"?
+     (whatever the bundle logs are...), see c#4</remark>
       For the Pacemaker log files, see the settings configured in the
-      <literal>logging</literal> section of &corosync.conf;. In case the
+      <literal>logging</literal> section of &corosync.conf;. <remark>taroth
```

Comment:
As mentioned in fate#324508, pacemaker no longer uses corosync logging settings.

Reply:
Removed this sentence.